    @Test
    public void canIssueCreateAndUpdateCallbacks() throws Exception {
        // Send a nodeDown
        sendNodeDownEvent(1);
        await().until(getNodeDownAlarmFor(1), hasSeverity(OnmsSeverity.MAJOR));
        assertThat(getNodeDownAlarmFor(1).call(), hasCounter(1));

        // Send another nodeDown
        sendNodeDownEvent(1);
        await().until(getNodeDownAlarmFor(1), hasCounter(2));
        assertThat(getNodeDownAlarmFor(1).call(), hasSeverity(OnmsSeverity.MAJOR));

        // Send a nodeUp
        sendNodeUpEvent(1);
        await().until(getNodeDownAlarmFor(1), hasSeverity(OnmsSeverity.CLEARED));
        await().until(getNodeUpAlarmFor(1), hasSeverity(OnmsSeverity.NORMAL));

        // Send another nodeDown
        sendNodeDownEvent(1);
        await().until(getNodeDownAlarmFor(1), hasSeverity(OnmsSeverity.MAJOR));
        assertThat(getNodeDownAlarmFor(1).call(), hasCounter(3));
    }

